en/news/_posts/2017-12-25-ruby-2-5-0-released.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: news_post
+title: "Ruby 2.5.0 Released"
+author: "naruse"
+translator:
+date: 2017-12-25 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.5.0.
+
+Ruby 2.5.0 is the first stable release of Ruby 2.5 series.
+It introduces many new features and performance improvements, for example:
```

Comment:
The line break and the `for example` at the end may not be needed.

Reply:
Please do wrap at about 80 chars, it‘s easier for translating (side-by-side diffs).

The „for example“ makes sense because only some improvements are listed here.